sdk/attestation/azure-security-attestation/azure/security/attestation/_administration_client.py

```diff
@@ -4,316 +4,689 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------
 
-from typing import List, Any, Optional, TYPE_CHECKING
+from typing import Dict, List, Any, Optional, TYPE_CHECKING, Tuple, Union
 
 from azure.core import PipelineClient
+from cryptography.hazmat.primitives import serialization
 from six import python_2_unicode_compatible
```

Comment:
Some of these imports don't seem to be used:
```suggestion
from typing import Dict, List, Any, TYPE_CHECKING, Tuple, Union

from cryptography.hazmat.primitives import serialization
```